content/pro/openbb-copilot.md

```diff
@@ -143,9 +171,49 @@ For example, you can ask OpenBB Copilot to fetch the latest stock price for a sp
 
 To query a custom endpoint, please explore the [Custom Backend](/pro/custom-backend/custom-backend) documentation.
 
-## Generating dashboard titles
+### Searching the Web
+
+The OpenBB Copilot has the ability to search the web for information that is not available in the data sources it has access to. This allows you to ask questions that go beyond the data displayed on your dashboard.
+
+To search the web, the user needs to be explicit in their query by adding the `@web` keyword in the query.
+
+<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/web_search_chat_example.jpg" alt="web search openbb copilot" width="80%" height="80%" />
+
+## Citations, Status Updates, and Artifacts
+
+When querying OpenBB Copilot, it is important to understand which data OpenBB
+Copilot used to answer your query. As a result, OpenBB Copilot will always cite the data
+source it referenced when responding to your query.
+
+<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/citation_chat_example.jpg" alt="citations openbb copilot" width="80%" height="80%" />
+
+We go one step further by providing live status updates which notify the user on actions taken by the OpenBB Copilot.
+For example, if the OpenBB Copilot is querying your uploaded data, it will provide a status update saying "Querying user files".
+
+### Artifacts
+
+A special kind of status update is the result artifact which has the final data used to answer the query. It either be a text or a table.
+The contents of the artifact are produced by the reasoning of the OpenBB Copilot.
+
+<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/artifact_chat_example.jpg" alt="result artifact openbb copilot" width="80%" height="80%" />
+
+To ensure transparency and extend the analysis, OpenBB Copilot will also return intermediate result artifacts that were used as reasoning steps to arrive to the final artifact and thus the answer.
+
+<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/intermediate_result_artifact_chat_example.jpg" alt="intermediate result artifact openbb copilot" width="80%" height="80%" />
+
+## Creating widgets from OpenBB Copilot responses
+
+Each time OpenBB Copilot generates a response, it will have an "Create widget from text" or "Create widget from table" button. Clicking this button will create a new widget on your dashboard based on the response.
```

Comment:
```suggestion
Each time OpenBB Copilot generates a response, it will have a "Create widget from text" or "Create widget from table" button. Clicking this button will create a new widget on your dashboard based on the response.
```